consonance-arch/src/main/java/io/consonance/arch/worker/WorkerRunnable.java

```diff
@@ -218,6 +234,78 @@ public void run() {
                 // Do the actual work
                 processJobMessage(workflowResult, statusJSON, job);
 
+                // create the job exchange
```

Comment:
Can you clarify if this block of code you added is actually redundant with 235 (processJobMessage())?